airtable.js

```diff
@@ -0,0 +1,52 @@
+// Don't forget to add your API key to the key variable
+const key = ""
+
+let table = base.getTable("Table 1");
+let view = table.getView('Needs Info');
+let result = await view.selectRecordsAsync({fields: ['Prompt']});
+for (let record of result.records) {
+    if (record) {
+        let prompt = await record.getCellValue('Prompt');
+        output.table(record.getCellValue('Prompt'));
+        output.text(prompt)
```

Comment:
Logging and debugging: The code currently uses output.text to log certain values. It's unclear where the output object comes from and how it's being used. Consider using proper logging techniques or the console for debugging purposes.